redbadger/website-honestly
# Fix a11y errors

### Changes

#### Description

This addresses several minor a11y issues that came up in audits done with the Webaim Wave tool and Google Chrome a11y tools.

Fixes  #718 and #587 

### Test

#### Test plan

- check for any styling regressions across the site
- check headings don't have missing levels (ie. h1 then h3) and that each page has one h1
